Resources/doc/reference/action_list.rst

```diff
@@ -142,8 +142,10 @@ Available types and associated options
 | array     |                | Displays an array                                                     |
 +-----------+----------------+-----------------------------------------------------------------------+
 | boolean   | ajax_hidden    | Yes/No; ajax_hidden allows to hide list field during an AJAX context. |
-+-----------+----------------+-----------------------------------------------------------------------+
-| boolean   | editable       | Yes/No; editable allows to edit directly from the list if authorized. |
++           +----------------+-----------------------------------------------------------------------+
+|           | editable       | Yes/No; editable allows to edit directly from the list if authorized. |
++           +----------------+-----------------------------------------------------------------------+
+|           | inverse        | Yes/No; reverses the background color (green for false, red for true) |
```

Comment:
That was actually on purpose. The logic was that in case one doesn't make sense, the other (hopefully) would. But I can change that if you think that it's a bad idea